docs/raiden_services.rst

```diff
@@ -0,0 +1,75 @@
+Raiden Services
+###############
+
+.. toctree::
+  :maxdepth: 3
+
+Introduction
+============
+
+The Raiden network includes auxiliary services, that enhance the overall user experience. These services include
+
+- finding short or cheap routes through the network and
+- allowing users to go offline safely by watching open payment channels.
+
+
+Pathfinding Services
+====================
+
+Pathfinding services having a global view on a token network can provide suitable payment paths for Raiden nodes. Raiden nodes can request paths for payments.
+
+The service will keep its view on the token network updated by listening to blockchain events and a public matrix room where current capacities and fees (Capacity Updates) are being published. Nodes can publish their channel capacities and fees in order to advertise their channels and mediate payments.
+
+
+Using a Pathfinding Service
+---------------------------
+
+Using a Pathfinding service increases the likelihood of successful payments, especially when multiple mediators are used. Therefore, using a Pathfinding service is enabled by default.
+
+.. note::
+  Direct token transfers (using no mediators) never need information from a PFS, so in that case no request for a PFS is done.
+
+To disable the usage of Pathfinding services use the ``--routing-mode`` command line flag. This setting is set to ``pfs`` by default, but can also be set to ``local`` or ``private`` if you don't want to use a Pathfinding service. See below for further information about privacy implications.
+
+
+Choosing a Pathfinding Service
+------------------------------
+
+By default the Raiden client will chose a suitable Pathfinding service for you. As all PFS' are registered in the service registry, it will iterate over all of them and chose one that has a fee which is lower then a given threshold.
```

Comment:
`chose` - >`choose`

Reply:
`PFS'` -> `PFSs`. No entirely sure this is correct, but PFS' imply plural AND genitive.